modules/ossm-migrating-reference-smcp-configurations.adoc

```diff
@@ -0,0 +1,448 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/reference/ossm-migrating-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-migrating-reference-smcp-configurations_{context}"]
+= Configuration fields mapping between Service Mesh 2 and Service Mesh 3
+
+Many of the `ServiceMeshControlPlane` `spec.` configuration fields in {SMProductName} 2 can be configured in the `Istio` resource in {SMProduct} 3.
+
+The following tables provide guidance for configuring your `Istio` resource in {SMProduct} 3.
+
+== Cluster configurations
+
+[cols="1,1"]
+|===
+| SMCP 2.6 configuration | Istio 3.0 configuration
+
+| spec.cluster.multiCluster.enabled
+| spec.values.global.multiCluster.enabled
+| spec.cluster.multiCluster.meshNetworks
+| spec.values.global.meshNetworks
+| spec.cluster.multiCluster.meshNetworks.endpoints
+| spec.values.global.meshNetworks.endpoints
+| spec.cluster.multiCluster.meshNetworks.endpoints.fromCID
+| spec.values.global.meshNetworks.endpoints.fromCidr
+| spec.cluster.multiCluster.meshNetworks.endpoints.fromRegistry
+| spec.values.global.meshNetworks.endpoints.fromRegistry
+| spec.cluster.multiCluster.meshNetworks.gateways
+| spec.values.global.meshNetworks.gateways
+| spec.cluster.multiCluster.meshNetworks.gateways.address
+| spec.values.global.meshNetworks.gateways.address
+| spec.cluster.multiCluster.meshNetworks.gateways.port
+| spec.values.global.meshNetworks.gateways.port
+| spec.cluster.multiCluster.meshNetworks.gateways.registryServiceName
+| spec.values.global.meshNetworks.gateways.registryServiceName
+| spec.cluster.name
+| spec.values.global.multiCluster.clusterName
+| spec.cluster.network
+| spec.values.global.network
+|===
+
+== General configurations
+
+[cols="1,1"]
+|===
+| SMCP 2.6 configuration | Istio 3.0 configuration
+
+| spec.general.logging.componentLevels
+| spec.values.global.logging.componentLevels
+| spec.general.logging.logAsJSON
+| spec.values.global.logAsJson
+| spec.general.validationMessages
+| spec.values.global.istiod.enableAnalysis
+|===
+
+== MeshConfig configurations
+
+[cols="1,1"]
+|===
+| SMCP 2.6 configuration | Istio 3.0 configuration
+
+| spec.meshConfig.discoverySelectors
+| spec.meshConfig.discoverySelectors
+| spec.meshConfig.extensionProviders
+| spec.meshConfig.extensionProviders
+|===
+
+== Mode configurations
+
+The mode configurations in the {SMProduct} 2 `ServiceMeshControlPlane` resource were:
+
+* Multitenant
+* Cluster-wide
+* Federation
+
```

Comment:
Little misunderstanding here. We still need the `In {SMProduct} 3, the mode is not configured using a single field in the `Istio` resource. By default the {SMProduct}3 control plane has access to all namespaces which is an equivalent for cluster-wide mode in {SMProduct} 2. A similar configuration to multitenant mode in {SMProduct} 2 can be achieved by configuring the `discoverySelectors` field. See "Deploying multiple service meshes on a single cluster".` Paragraph here, it can be standard paragraph not a note.